sale_layout_category_hide_detail/README.rst

```diff
@@ -117,6 +124,8 @@ Contributors
   * Ernesto Tejeda
   * Pedro M. Baeza
 
+* Druidoo (<https://www.druidoo.io>)
+
 Maintainers
 ~~~~~~~~~~~
 
```

Comment:
Changes in `README.rst` will be overwritten by the auto-readme generation tool.

Make sure you make all changes in the files located at `/readme`.
Changing `README.rst` will have no impact after the pr is merged, you can leave it unchanged or not, but make sure all the changes are in the separate files